### DISCUSSION TITLE
Disable F2 shortcut?

### DISCUSSION THREAD

**Discussion by Solarunit** [Taiko2k/Tauon#739] [Ideas]
I think that F2 shortcut (next theme) is not that needed. You usually have 2-3 favorite themes and could easily change them in the settings. Meanwhile it affects other things: 

- I often accidentally  press F2 and have to apply my favorite theme again, 
- Sometimes I want to rename playlist by pressing F2 (because it's a common shortcut to rename things) instead I see that my theme is gone.

**Comment by fahim2690** [comment]
@Solarunit You can unbind that in Tauon's input.txt (in Tauon's userdata folder, which you can access either by navigating to it if you're using a dev install or from 
![image](https://user-images.githubusercontent.com/37258115/164507486-0b928fb9-ea1e-4e8c-8037-a1b23913bd09.png)
), it's listed as "cycle-theme"

**Reply by Solarunit** [reply]
Good idea, done that!
`# cycle-theme F2`

BTW what do you mean by "if you're using a dev install"? :thinking:

**Reply by fahim2690** [reply]
Dev install == running it straight off of the git repo, not in packaged form (or managed by any package manager). The folder locations are different w/ a dev install because it'll shove everything under your installation folder instead of the relevant XDG directories (or flatpak'd versions thereof) so e.g. your user-data folder would be under /<repo_path>/user-data.

**Reply by Solarunit** [reply]
I got it, ty.
